src/main/java/racingcar/domain/Winners.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+    private List<RacingCar> winners;
```

Comment:
일급 컬렉션인 `RacingCars`를 사용해 보세요.

```suggestion
    private RacingCars winners;
```